lessons/students/3_classes_and_beyond/17_classes/17_classes.py

```diff
@@ -47,38 +55,40 @@ def x_plus_five(self): # This is a method. Methods always take at least one para
 
 ####################################################################################################
 
-# TITLE: Section 1.2 - The __init__ Function
+# TITLE: Section 1.2 - The "__init__" Method
 
 # The "__init__()" function is a special Python function that is called automatically by an object
 # creation statement. In OOP, the "__init__()" function is known as a constructor, as its job is to
 # build (construct) an object of the type specified by the class. Below we are defining the class
-# "Cat" and want to give it instance attributes of "name" and "color". Remeber that instance
+# "Cat" and want to give it instance attributes of "name" and "color". Remember that instance
 # attributes are defined in each individual class instantiation.
 
 class Cat:
   animal_type = 'feline' # This is a class attribute that will be shared by all instances of "Cat".
 
-  def __init__(self, name, color): # Our constructor takes 3 parameters: "self" (the instance of
-                                   # the class), name, and color.
-    # Using the syntax "self.var", where "var" is a given parameter, we can set attributes of a
-    # given instantiation of a class during its creation.
+  # Our constructor takes 3 parameters: "self" (the instance of the class), name, and color.
+  def __init__(self, name, color):
+    """
+    Using the syntax "self.var", where "var" is a given parameter, we can set
+    attributes of a given instantiation of a class during its creation.
+    """
     self.name = name
     self.color = color
 
-# Below we are intantiating a class of "Cat" and setting the "name" attribute to "Simba" and the
+# Below we are instantiating a class of "Cat" and setting the "name" attribute to "Simba" and the
 # "color" attribute to "orange".
 simba = Cat("Simba", "orange")
 grumpy = Cat("Grumpy Cat", "brown and white")
 
-# Again, we can use dot notation to access the attributes of "simba" and "grumpy".
+# Again, we can use dot notation to access the attributes of "Simba" and "Grumpy Cat".
```

Comment:
Was referencing variable names here. Suggestion:
1. Keep var names lowercase
2. change "grumpy" -> "grumpy_cat"

We are trying to access attributes of a class instantiation, not specifically the "name" attributes"